quickstart/aspnet/ContosoTeamStats/App_Start/BundleConfig.cs

```diff
@@ -8,8 +8,10 @@ public class BundleConfig
         // For more information on bundling, visit https://go.microsoft.com/fwlink/?LinkId=301862
         public static void RegisterBundles(BundleCollection bundles)
         {
-            bundles.Add(new ScriptBundle("~/bundles/jquery").Include(
-                        "~/Scripts/jquery-{version}.js"));
+            var jqueryCdn = "https://ajax.aspnetcdn.com/ajax/jQuery/jquery-3.3.1.js";
```

Comment:
can we just inline this string on line 13 rather than declare a variable for it?